mycroft/skills/intent_services/padatious_service.py

```diff
@@ -52,6 +52,11 @@ def __init__(self, bus, config):
         self.bus.on('detach_intent', self.handle_detach_intent)
         self.bus.on('detach_skill', self.handle_detach_skill)
         self.bus.on('mycroft.skills.initialized', self.train)
+        self.bus.on('intent.service.padatious.get', self.handle_get_padatious)
+        self.bus.on('intent.service.padatious.manifest.get',
+                    self.handle_manifest)
+        self.bus.on('intent.service.padatious.entities.manifest.get',
+                    self.handle_entity_manifest)
```

Comment:
Don't these duplicate the bus handlers registered in [intent_service.pyL129](https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/skills/intent_service.py#L129)?

They should be removed as well if these are moved here.